### ISSUE TITLE
Trigger items in "video/Image bin" in Propresenter

### ISSUE THREAD

**Issue by shaunar2006** [bitfocus/companion#748]
Currently, it is only possible to trigger specific slides in a playlist. It is impossible to trigger slides from the video/image bin.

This is useful to trigger different backgrounds in propresenter without having to put the backgrounds inside a playlist just to trigger them. Even when putting the backgrounds in a separate playlist, propresenter navigates away from your current playlist (containing the foreground/lyrics) and opens the playlist with the backgrounds, which is inconvienient.

![Screen Shot 2019-07-23 at 12 40 39 PM](https://user-images.githubusercontent.com/53232897/61734437-89effb00-ad47-11e9-9782-ada88b5a0be6.png)

**Comment by cgarwood** [comment]
Companion uses the same network API the the ProRemote and Stage Display mobile apps use. I don't believe there is support in the API to trigger things from the video bin unless they're already on a slide.

**Comment by shaunar2006** [comment]
Ah. That makes sense. 
Is there a way to trigger midi notes from companion? Using midi, it is possible to trigger those slides.

**Comment by josephdadams** [comment]
The “midi relay” module I wrote for Companion, paired with the midi relay server component, could help you with this, @shaunar2006.

**Comment by shaunar2006** [comment]
Joseph Adams, Could you possibly walk me through this process? Using a mac.

**Comment by josephdadams** [comment]
Sure thing, it's pretty easy. Install this on your ProPresenter machine and run it: https://github.com/josephdadams/midi-relay/releases
It runs in the command line.

Use Companion 1.4 or 2.0 and search for the "midi relay" module. If it's not included, let me know and we can try to get it into a build. The config settings for the instance of this module need to include the IP address of the ProPresenter computer. It runs on port 4000.

Set up a new button action to send MIDI Note On and specify the MIDI port you are using with ProPresenter from the list. Send the note you've configured ProPresenter to listen for. (I'm just assuming you know how to do the ProPresenter MIDI side, if not, let me know.)

Basically this program sends HTTP requests from Companion to the computer running MIDI Relay, where it is then turned into a MIDI command and relayed locally on the port you chose.

**Comment by shaunar2006** [comment]
Thank You! This works Perfectly!!

**Comment by josephdadams** [comment]
Closing because this is only possible through MIDI control, and this can be done using the MIDI Relay module as mentioned above.

**Comment by shaunar2006** [comment]
Sorry, I need to open this again... 
I tested the midi relay module on my personal macbook and it seemed to work fine. The midi module is only available on Companion 2.0, not 1.3.
